From eb2cfc6ebbd4b7a8785bf43f7f72fa3023683b89 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Thu, 9 Jun 2016 11:16:11 +0300
Subject: [PATCH 1189/1240] fix: Increase the u-boot flash allocation to 2MB

- Due to increased size of the u-boot image it is required
  to change the u-boot flash region size from 1MB to 2MB
  effectively moving u-boot environment to 2MB boundary.

Change-Id: I671c7025908540e4be1b15ca2eb021d86c9d30ad
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30387
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 include/configs/mvebu-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mvebu-common.h b/include/configs/mvebu-common.h
index edeb425..f6de82f 100644
--- a/include/configs/mvebu-common.h
+++ b/include/configs/mvebu-common.h
@@ -66,7 +66,7 @@
 #define CONFIG_SYS_INIT_SP_ADDR         (CONFIG_SYS_TEXT_BASE + 0xFF0000)   /* End of 16M scrubbed by training in bootrom */
 #define CONFIG_SYS_GBL_DATA_SIZE        128	/* Size in bytes reserved for init data */
 #define CONFIG_SYS_MAXARGS		32	/* Max number of command argg */
-#define CONFIG_UBOOT_SIZE		0x100000 /* 1MB */
+#define CONFIG_UBOOT_SIZE		0x200000 /* 2MB */
 #undef  CONFIG_USE_IRQ
 
 /* Memory reserve */
-- 
1.9.1

